updateStableIds/pom.xml

```diff
@@ -57,4 +57,21 @@
 	</repository>
   </repositories>
   
+  <build>
+	  <plugins>
+	    <plugin>
+	      <artifactId>maven-assembly-plugin</artifactId>
+	      <configuration>
+	        <archive>
+	          <manifest>
+	            <mainClass>org.reactome.release.update_stable_ids.Main</mainClass>
```

Comment:
The Java convention is that package names do not contain underscores, unless they are a replacement for a hyphen/dash in a domain name. So, the package `update_stable_ids` would normally be written as `updatestableids` or `updateStableIds`.

https://stackoverflow.com/a/3179277 - with links to other references on package name conventions.